### PR TITLE
Update 5.0 repo name to "nightly" instead of "core-nightly"

### PR DESCRIPTION
**Pull request by mthalman**
Related to #1765

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1787,7 +1811,15 @@
               "variant": "v8"
             }
           ]
-        },
+        }
+      ]
+    },
+    {
+      "id": "sdk-preview",
```
Comment (MichaelSimons) [inline manifest.json:1818]: Is it reasonable to name items like the id, readmePath, and template in durable terms that won't change once 5.0 is RTM?  If we used the terms like `sdk` and `core-sdk` we would have long term stable names that won't change.

File: README.aspnet.preview.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+docker run -it --rm -p 8000:80 --name aspnetcore_sample mcr.microsoft.com/dotnet/core/samples:aspnetapp
+```
+
+After the application starts, navigate to `http://localhost:8000` in your web browser.
+
+See [Hosting ASP.NET Core Images with Docker over HTTPS](https://github.com/dotnet/dotnet-docker/blob/master/samples/host-aspnetcore-https.md) to use HTTPS with this image.
+
+# Related Repos
+
+.NET Core:
```
Comment (MichaelSimons) [inline README.aspnet.preview.md:43]: Shouldn't the `core` terminology be eliminated in the new .NET readmes except for where 3.1 and older versions are referenced?

File: README.aspnet.preview.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+```console
+docker run -it --rm -p 8000:80 --name aspnetcore_sample mcr.microsoft.com/dotnet/core/samples:aspnetapp
+```
+
+After the application starts, navigate to `http://localhost:8000` in your web browser.
+
+See [Hosting ASP.NET Core Images with Docker over HTTPS](https://github.com/dotnet/dotnet-docker/blob/master/samples/host-aspnetcore-https.md) to use HTTPS with this image.
+
+# Related Repos
```
Comment (MichaelSimons) [inline README.aspnet.preview.md:41]: What are you thinking for the UX for relating/referring users between the .NET and .NET Core repos?  We should have messaging that helps provide clarity between the various repos.

File: manifest.json (hunk 1 above)
Comment (mthalman) [reply manifest.json]: Makes senses.  Fixed.

File: README.aspnet.preview.md (hunk 2 above)
Comment (mthalman) [reply README.aspnet.preview.md:45]: Fixed

File: README.aspnet.preview.md (hunk 3 above)
Comment (mthalman) [reply README.aspnet.preview.md:43]: This is definitely tricky.  I've proposed some changes in the README.aspnet.md and README.aspnet.preview.md files.  Let me know what you think and I can roll those out to the rest of the readmes.

File: README.aspnet.preview.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+.NET Core:
+
+* [dotnet/core](https://hub.docker.com/_/microsoft-dotnet-core/): .NET Core
+* [dotnet/core/samples](https://hub.docker.com/_/microsoft-dotnet-core-samples/): .NET Core Samples
+
+.NET Core (Preview):
```
Comment (MichaelSimons) [inline README.aspnet.preview.md:50]: I lean towards it is only necessary to link to the product family here.  The pattern is to only link to the individual repos that are part of the same product family.

File: README.aspnet.preview.md (hunk 3 above)
Comment (MichaelSimons) [reply README.aspnet.preview.md:43]: I agree it is tricky.  I think this is looking good.  It will help once 5.0 is shipped and not in "preview".

**Comment by mthalman** [comment]
I'll address the remaining review comments in a separate PR.  We're wanting to get this in now to jumpstart the process of onboarding to Docker Hub.

**Pull request merged**